Test/Test.cs

```diff
@@ -146,164 +46,480 @@ public void InitClient()
+            var outputSchema = schema;
+            var response = Toby.CreateTransition(transitionType, inputSchema, outputSchema, name, description);
+            CheckKeys(new [] {"name", "transitionId", "transitionType"}, response);
+        }
 
-            JObject jsonResponse = JObject.Parse(response.ToString());
-            var predictionString = jsonResponse["predictions"].ToString();
-            var predictions = JsonSerialPublisher.DeserializeObject<List<Dictionary<string, Dictionary<string, object>>>>(predictionString);
-            foreach (var line in predictions)
-            {
-                Console.WriteLine("\n New Line Found");
-                foreach (var field in line)
-                {
-                    Console.WriteLine($"{field.Key}: {field.Value["val"]}, confidence: {field.Value["confidence"]}");
-                }
-            }
+        [TestCase("docker")]
+        [TestCase("manual")]
+        [TestCase(null)]
+        public void TestListTransitions(string? transitionType) {
+            var response = Toby.ListTransitions(transitionType);
+            CheckKeys(new [] {"transitions"}, response);
+        }
+
+        public void TestListTransitions() {
+            var response = Toby.ListTransitions(new List<string>{"docker", "manual"});
+            CheckKeys(new [] {"transitions"}, response);
+        }
+
+        public void TestGetTransitionExecution() {
+            var executionId = $"las:transition-execution:{Guid.NewGuid().ToString()}";
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var response = Toby.GetTransitionExecution(transitionId, executionId);
+            CheckKeys(new [] {"transitionId", "executionId", "status"}, response);
+        }
+
+        [TestCase("foo", "bar")]
+        [TestCase(null, null)]
+        public void TestUpdateTransition(string name, string description) {
+            var schema = new Dictionary<string, string>() {
+                {"schema", "https://json-schema.org/draft-04/schema#"},
+                {"title", "response"}
+            };
+            var inputSchema = schema;
+            var outputSchema = schema;
+            var transitionId = $"las:transition:{Guid.NewGuid().ToString()}";
+            var response = Toby.UpdateTransition(transitionId, inputSchema, outputSchema, name, description);
+            CheckKeys(new [] {"name", "transitionId", "transitionType"}, response);
+        }
+
+        [Test]
+        public void TestExecuteTransition() {
+            var transitionId = $"las:transition-execution:{Guid.NewGuid().ToString()}";
+            var response = Toby.ExecuteTransition(transitionId);
+            CheckKeys(new [] {"transitionId", "executionId", "status"}, response);
+        }
+
+        [TestCase(
+            "running",
+            "las:transition-execution:08b49ae64cd746f384f05880ef5de72f",
+            3,
+            null,
+            "completedBy",
+            "asc"
```

Comment:
"ascending" or "descending" are the only valid inputs.

Reply:
Why not allow "asc" and "desc"? It's more typical.

Reply:
Either way, it should be made a enum in the oas file.